tools/scylla-nodetool.cc

```diff
@@ -3742,6 +4000,165 @@ For more information, see: {}"
                 table_stats_operation
             }
         },
+        {
+            {
+                "tasks",
+                "Manages task manager tasks",
+                "",
+                { },
+                {
+                    typed_option<sstring>("command", "The uuid of a task", 1),
+                },
+                {
+                        {
+                            "abort",
+                            "Aborts the task",
+fmt::format(R"(
+Aborts a task with given id. If the task is not abortable, the right message
+is printed.
```

Comment:
This is a bit obscure, what is the "right message"? You mean the "appropriate message will be printed, depending on why the abort failed"?

You should document the exit codes too.

Reply:
Regarding the exit codes: do we want to exit with 0 and "task not abortable" message or should an exit code signal it?

Reply:
I think it is fine if non-zero exit code are only used with `--quiet`.